Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -8385,6 +8386,16 @@ void HTMLMediaElement::setOverridePreferredDynamicRangeMode(DynamicRangeMode mod
     player->setShouldDisableHDR(shouldDisableHDR());
 }
 
+void HTMLMediaElement::setPlatformDynamicRangeLimit(PlatformDynamicRangeLimit platformDynamicRangeLimit)
+{
+    m_platformDynamicRangeLimit = platformDynamicRangeLimit;
```

Comment:
Why do we store this property here? I do see it used in `HTMLMediaElement`.

Reply:
Good eyes!
I need it in the next patch where it will be combined with an external notification.
But I suppose I could move this to the other patch, just in case we change tactics there...

Reply:
Actually it's used at line 7951, when a player is created after this call here, so that we can give the new player the correct DRLimit.